### DISCUSSION TITLE
Shelly Mini1G3 OTA URL

### DISCUSSION THREAD

**Discussion by dippes** [arendst/Tasmota#22406] [Support request]
Hello

I have noticed that after an update the wrong OTA URL is always stored in the WIF.

Instead of ‘ http://ota.tasmta.com/tasmota32/release/tasmota32oc3.bin’

this ‘ota.com/tasmota32/release/tasmota32-EN.bin’

which leads to a save boot.

Is this normal or does the ota url have to be changed manually before an update?

**Comment by Noschvie** [comment]
You mean: http://ota.tasmota.com/tasmota32/release/tasmota32c3.bin
